perception/navigator_lidar_oa/src/ConnectedComponents.h

```diff
@@ -182,10 +185,9 @@ std::vector< std::vector<int> > ConnectedComponents(OccupancyGrid &ogrid, std::v
 	}
 
 	//Re-organize obstacles, how many connections to count as an obstacle?
-	int newId = 0;
 	objects.clear();
 	for (auto ii : mapObjects)  {
-		if (ii.second.strikesPersist.size() >= 10) {
```

Comment:
Good with me. How often should a node check for possible changes from a rosparam?